aspnetcore/grpc/browser.md

```diff
@@ -75,6 +75,26 @@ Traditional gRPC over HTTP/2 supports streaming in all directions. gRPC-Web offe
 
 When using gRPC-Web, we only recommend the use of unary methods and server streaming methods.
 
+### HTTP protocol
+
+The ASP.NET Core gRPC service template included with .NET creates an app that's configured for HTTP/2 only. This is a good default when an app only supports traditional gRPC over HTTP/2. gRPC-Web, however, works with both HTTP/1.1 and HTTP/2. Some platforms, such as UWP or Unity, can't use HTTP/2. To support all client apps, configure the server to enable HTTP/1.1 and HTTP/2.
```

Comment:
Two options to deal with "included" because it tricks the reader into thinking it's the verb for "template":

* Move the phrase to the front of the sentence:

  ```suggestion
  Included with .NET, the ASP.NET Core gRPC service template creates an app that's only configured for HTTP/2. This is a good default when an app only supports traditional gRPC over HTTP/2. gRPC-Web, however, works with both HTTP/1.1 and HTTP/2. Some platforms, such as UWP or Unity, can't use HTTP/2. To support all client apps, configure the server to enable HTTP/1.1 and HTTP/2.
  ```

* Set it off with commas:

  ```suggestion
  The ASP.NET Core gRPC service template, included with .NET, creates an app that's only configured for HTTP/2. This is a good default when an app only supports traditional gRPC over HTTP/2. gRPC-Web, however, works with both HTTP/1.1 and HTTP/2. Some platforms, such as UWP or Unity, can't use HTTP/2. To support all client apps, configure the server to enable HTTP/1.1 and HTTP/2.
  ```